content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "Production environment"
+description: Create a production-quality Kubernetes cluster
+weight: 30
+no_list: true
+---
+<!-- overview -->
+
+A production-quality Kubernetes cluster requires planning and preparation.
+If co-workers and customers depend on Kubernetes being available to run critical
+workloads, it is your job to make sure that the platform is set up and
+maintained to meet their demands.
+
+<!-- body -->
+
+## Production issues
+
+A personal Kubernetes learning or development environment is not meant to be
+used in production. A production environment may require secure access by many
+users, consistent availability, and the resources to adapt to changing demands.
+
+As you decide where you want your production Kubernetes environment to live
+(on premises or in a cloud) and the amount of management you want to take
+on or hand to others, consider how your requirements for a Kubernetes cluster
+are colored by the following issues:
```

Comment:
```suggestion
are influenced by the following issues:
```
Nit on word choice. Maybe `influenced` or `impacted` over `colored`

Reply:
I see that my language was too colorful. :-) Okay. Influenced it is.